config-example.template.conf

```diff
@@ -0,0 +1,24 @@
+set -a
+
+# The authentication provider's endpoint for retrieving a session token
+TARGET_SSO="token-endpoint-url"
+
+# The QA environment is used by default, but others can be targeted
+# instead by setting (and enabling) the following value...
+#TARGET_API="https://api.<env-name>.notprod.lev.homeoffice.gov.uk"
+
+# O-Auth 2 client details
+CLIENT_ID="client-id"
+CLIENT_SECRET="client-secrert"
+
+# O-Auth 2 username and password
+USER_NAME="user@domain"
+PASSWORD='*********'
+
+# Certificate and key files for mutual TLS
+CLIENT_CERT="./config/client.crt"
+CLIENT_KEY="./config/private.key"
```

Comment:
Should we add something to `.gitignore`?
```
*.key
```
?

Reply:
That whole folder `./config` is already ignored

Reply:
Okay.